jwst/extract_1d/extract.py

```diff
@@ -274,26 +198,27 @@ def get_extract_parameters(
         This argument must be positive or zero, and it is only used if
         background regions have been specified.
 
-    use_source_posn : bool or None
+    use_source_posn : bool or None, optional
         If True, the target and background positions specified in `ref_dict`
         (or a default target position) will be shifted to account for
         the actual source location in the data.
         If None, the value specified in `ref_dict` will be used, or it will
         be set to True if not found in `ref_dict`.
 
+    subtract_background : bool or None, optional
```

Comment:
I think None needs to remain an option, so that it can decide based on the input data whether it should be turned on or not.  

At the moment, it could probably default to True, since it gets set to False if there are no background regions available anyway, but when optimal extraction becomes an option, it will be possible to subtract background without specifying a background region, so it will need to distinguish between True, False, and None at that point.